ament_cmake_core/cmake/uninstall_target/ament_cmake_uninstall_target.cmake.in

```diff
@@ -29,7 +29,7 @@ function(ament_cmake_uninstall_target_remove_empty_directories path)
 endfunction()
 
 # uninstall files installed using the standard install() function
-set(install_manifest "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+set(install_manifest "@CMAKE_BINARY_DIR@/install_manifest.txt")
```

Comment:
This seems like an improvement 👍 .

I know we don't normally support this, but if someone were using fetch-content style superbuilds, would it be a problem if multiple projects installed their manifests to the same place? If so, would `PROJECT_BINARY_DIR` be a better fit? 

https://cmake.org/cmake/help/latest/variable/PROJECT_BINARY_DIR.html

Reply:
Even if you add via FetchContent dependencies that install files, the file that they install will be added during the installation process to the same (global) `install_manifest.txt` file located in `CMAKE_BINARY_DIR` (I just tested to be sure with https://github.com/eliasdaler/cmake-fetchcontent-tutorial-code).

Reply:
Ah! I also misunderstood that this file is being generated by CMake, and we just need the correct path to read it here.

Thanks for testing the FetchContent case!